src/SmartTransactionsController.ts

```diff
@@ -138,7 +139,9 @@ export type SmartTransactionsControllerGetStateAction =
 export type SmartTransactionsControllerActions =
   SmartTransactionsControllerGetStateAction;
 
-export type AllowedActions = NetworkControllerGetNetworkClientByIdAction;
```

Comment:
Good idea to remove these exports, but we should document this as a breaking change as well